### PR TITLE
[WIP] Update doc

### PR DESCRIPTION
**Pull request by DhritiShikhar**
### Motivation

[1] Updates doc according to new annotation format
[2] Fixes https://github.com/redhat-developer/service-binding-operator/issues/824

### CODE REVIEW

**Review by sbose78 — APPROVED** [review]

**Comment by sbose78** [comment]
Taking this in, in spite of the failed test because this a docs only PR, and the docs fix is critical :)

File: docs/SBO User Guide.md [hunk 1]
```diff
@@ -688,7 +688,7 @@ spec:
     Annotation
 
     ```
-    “servicebinding.dev/uri: "path={.status.data.connectionURL}”
+    “service.binding/uri: "path={.status.data.connectionURL}”
```
Comment (sbose78) [inline docs/SBO User Guide.md:691]: should be "service.binding/uri" ?

Comment (DhritiShikhar) [reply docs/SBO User Guide.md]: Fixed

**Comment by DhritiShikhar** [comment]
@redhat-developer/service-binding Can we merge this PR?

**Comment by pedjak** [comment]
It looks that https://github.com/redhat-developer/service-binding-operator/pull/829 contains the similar changes. cc @Avni-Sharma

**Comment by DhritiShikhar** [comment]
Closing this PR in favour of #829

**Pull request closed without merging**